qnexus/models/filters.py

```diff
@@ -216,7 +216,14 @@ class JobStatusFilter(BaseModel):
     """Job status filter"""
 
     status: list[JobStatusString] | None = Field(
-        default=["COMPLETED", "QUEUED", "SUBMITTED", "RUNNING", "CANCELLED", "ERROR"],
+        default=[  # type: ignore
+            "COMPLETED",
+            "QUEUED",
+            "SUBMITTED",
+            "RUNNING",
+            "CANCELLED",
+            "ERROR",
+        ],
```

Comment:
These were getting flagged by MyPy all of a sudden for being the wrong type (str vs Literal["foo"]), couldn't think of a cleaner way to fix the issue other than just ignore it.